src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/TrackingOptInModal.java

```diff
@@ -0,0 +1,172 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.WikiaWebDriver;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.core.networktrafficinterceptor.NetworkTrafficInterceptor;
+import com.wikia.webdriver.common.core.url.Page;
+import com.wikia.webdriver.common.core.url.UrlBuilder;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+
+import org.openqa.selenium.Cookie;
+import org.openqa.selenium.TimeoutException;
+import net.lightbody.bmp.core.har.HarEntry;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.time.Duration;
+import java.util.List;
+
+public class TrackingOptInModal extends BasePageObject {
+
+    @FindBy(css = "body > div:nth-child(5) > div > div")
+    WebElement modal;
+
+    @FindBy(css = "div._36mdTyonPU0bxbU70dDO4f")
```

Comment:
Aren't this classes auto-generated? If yes, than this class will be change with every build & release.